test/chunked_loss/test_cpo_loss.py

```diff
@@ -126,7 +126,7 @@ def test_correctness(
         input1, weight1, target, bias1, alpha=alpha
     )
     loss2 = LigerFusedLinearCPOFunction.apply(
-        input2, weight2, target, bias2, ignore_index, beta, alpha, True
+        input2, weight2, target, bias2, ignore_index, beta, alpha, False
```

Comment:
Why are we changing the test case for an unrelated alignment algo?

Reply:
Sorry my bad.